src/layouts/base.njk

```diff
@@ -58,90 +58,93 @@
       </div>
     </section> 
         
-      <section id="projects" class="projects">
-        <h2>Projects</h2>
-        <div class="grid">
+      <section id="projects" class="cmp-projects">
+        <div class="cmp-projects__container">
+          <h2 class="cmp-projects__heading">Projects</h2>
 
-        <!-- neverland travelers -->
-        <div class="neverland-travelers">
-          <div class="left">
-            <div class="nt-text">
-              <h3>
-                Website for personal vlog
-              </h3>
-              <p>
-                I built and deployed the first version of a website dedicated to my personal vlog channel called Neverland Travelers. 
-              </p>
-              <div class="links">
-                <a href="work/neverland-travelers.html" class="link1">Learn More</a>
-                <a href="https://www.neverlandtravelers.com" target="_blank" class="link2">
-                  See Live Site &nbsp; &nbsp;  
-                  {% image 'src/public/usedresources/external-link.svg', '', 'Visit Neverland Travelers site' %}
-                </a>
+          <div class="obj-projects-grid">
+            {# neverland travelers vlog #}
+            <div class="obj-projects-grid__item obj-projects-grid__item--vlog">
+              {# text container on left #}
+              <div class="obj-projects-grid__item-text">
+                <h3 class="obj-projects-grid__heading">
+                  Website for personal vlog
+                </h3>
+                <p class="obj-projects-grid__description">
+                  I built and deployed the first version of a website dedicated to my personal vlog channel called Neverland Travelers. 
+                </p>
+                <div class="obj-projects-grid__links">
+                  <a href="/work/neverland-travelers.html" class="cmp-button cmp-button--primary" aria-label="Learn more about the Neverland Travelers vlog project site">Learn More</a>
+                  <a href="https://www.neverlandtravelers.com" target="_blank" class="cmp-button cmp-button--secondary">
+                    Visit Site
+                    {% image 'src/public/usedresources/external-link.svg', 'cmp-button__icon', 'Visit Neverland Travelers site' %}
```

Comment:
If you look at the accessibility tree, you can see how the accessible name for this link is resolved to "Visit Site Visit Neverland Travelers site", which isn't ideal for screen reader users. 

![Screenshot 2023-12-08 at 2 42 20 PM](https://github.com/marissahuysentruyt/marissahuysentruyt/assets/82047862/8e36a2a3-2b4e-49f4-8ba9-5c79ee428c79)

There's also a visual indicator that the link will open in a new tab, but screen reader users wouldn't know to expect that behavior.

I tend to avoid `target="_blank"` if I can, since I figure if users want to open a link in a new tab, they will, but you could set the alt text to an empty string and set `aria-label="Visit Neverland Travelers site (opens in a new tab)"` on the link, which is probably the second easiest way to set the accessible name.

The first easiest way (in terms of code) to is to make the accessible name the same as the visual text, but that would require some redesign effort.

Reply:
I haven't used the accessibility tree much, if at all really. This is a good reminder to start!

I've been incorrectly using `aria-label` stuff during some of our project work, so Catherine helped me understand that `aria-label` is used to replace any visible text. I think however, I'm going to wait on accessibility improvements until the next pass, for now at least.

Reply:
I did remove the `target="_blank". https://github.com/marissahuysentruyt/marissahuysentruyt/pull/26/commits/bac56a9dbd1fff1b2886abe18574173e49cb095f